torch/onnx/symbolic_opset11.py

```diff
@@ -894,6 +894,21 @@ def embedding_bag(g,
     return loop.node().output(), None, None, None
 
 
+@parse_args('v', 'v', 'f', 'f')
+def embedding_renorm(g, weight, indices, max_norm, norm_type):
+    unique_indices = g.op("Unique", indices)
+    partial_weight = g.op("Gather", weight, unique_indices)
+    norm_type = "ReduceL2" if int(norm_type) == 2 else "ReduceL1"
```

Comment:
PyTorch seems to accept 0 and 3 as valid norm types. I guess we only support 1 and 2?
If so let's make that explicit and raise an error if norm type is not 1 or 2.

Reply:
Good catch, added check to raise when not 1 or 2.